system-tests/test/minting-unminting.test.ts

```diff
@@ -176,12 +220,17 @@ describe("System Test - Minting and unminting", () => {
         // Since the reveal deposit logic does not perform SPV proof, we
         // can reveal the deposit transaction immediately without waiting
         // for confirmations.
-        await TBTC.revealDeposit(
-          depositUtxo,
-          deposit,
-          electrumClient,
-          depositorBridgeHandle,
-          deposit.vault
+        const rawDepositTransaction = await electrumClient.getRawTransaction(
+          depositUtxo.transactionHash
+        )
+        const depositRawTxVectors = extractBitcoinRawTxVectors(
+          rawDepositTransaction
+        )
+        await depositorBridgeHandle.revealDeposit(
+          depositRawTxVectors,
+          depositUtxo.outputIndex,
+          depositReceipt,
+          EthereumAddress.from(tbtcVault.address) // Use TBTCVault as target vault.
         )
```

Comment:
All deposit creation and minting should be done in the new fashion: https://docs.threshold.network/app-development/tbtc-v2/tbtc-sdk/guides/deposit-and-mint

Reply:
Done in 5691a18c4dff48f0fcfc198c32e245ed28903472.